RackSDK.cmake

```diff
@@ -100,8 +100,9 @@ file(INSTALL ${PLUGIN_DISTRIBUTABLES} DESTINATION ${PLUGIN_NAME})
 
 # A quick installation target to copy the plugin library and plugin.json into VCV Rack plugin folder for development.
 # CMAKE_INSTALL_PREFIX needs to point to the VCV Rack plugin folder in user documents.
+set (PLUGIN_SUFFIX $<IF:$<BOOL:$<TARGET_PROPERTY:${RACK_PLUGIN_LIB},SUFFIX>>,$<TARGET_PROPERTY:${RACK_PLUGIN_LIB},SUFFIX>,${CMAKE_SHARED_MODULE_SUFFIX}>)
```

Comment:
![](https://i.giphy.com/media/ZQBH5pvoyiZSE/giphy.gif)

Reply:
Yeah there must be a better way
Cmake knows the full path of the target I just need to dig up the property name. We use it in some of the surge installer code